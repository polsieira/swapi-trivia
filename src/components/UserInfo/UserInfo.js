import React from 'react';

const UserInfo = () => {
  return(
    <>
      <h2>Name: {}</h2>
      <p>Quote: {}</p>
      <p>Ranking: {}</p>
      <button>Sign Out</button>
    </>
  )
}

export default UserInfo;