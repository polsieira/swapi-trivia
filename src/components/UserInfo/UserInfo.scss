$star-wars-yellow: #FFE81F;

.user-info {
  color: $star-wars-yellow;
  padding: 2% 1% 0 85%;
}

.user-info-p {
  margin: 3px 0;
}

.button--sign-out {
  border: $star-wars-yellow 1px solid;
  font-size: 15px;
  font-weight: bold;
  height: 25px;
  background: none;

  color: $star-wars-yellow;
}