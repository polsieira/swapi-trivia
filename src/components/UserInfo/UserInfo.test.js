import React from 'react';
import { shallow } from 'enzyme';

import UserInfo from './UserInfo';

describe('UserInfo', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<UserInfo
      name='Pol'
      quote='I have a dream'
      ranking='Scoundrel'
    />)
  });

  it('should match the snapshot with all data passed correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });


});