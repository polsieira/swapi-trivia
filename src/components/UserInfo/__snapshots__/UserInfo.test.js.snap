// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo should match the snapshot with all data passed correctly 1`] = `ShallowWrapper {}`;
